zipline-gradle-plugin/src/test/projects/basic/build.gradle.kts

```diff
@@ -0,0 +1,25 @@
+buildscript {
+  repositories {
+    maven {
+      url = file("$rootDir/../../../../../build/testMaven").toURI()
+    }
+    mavenCentral()
+    google()
+  }
+  dependencies {
+    println("ZIPLINE VERSION = " + project.property("ziplineVersion"))
+    classpath("app.cash.zipline:zipline-gradle-plugin:${project.property("ziplineVersion")}")
+    classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:1.6.10")
```

Comment:
You can import the root project version catalog to keep dependencies in these fixture projects in sync with the enclosing project.

https://github.com/cashapp/redwood/blob/c8454d716119b50e80ae44da6fc09a7529727463/redwood/redwood-gradle-plugin/src/test/fixture/compose-ui/settings.gradle#L7-L13

Reply:
Oooooh yes that’s what I need.

Reply:
Just realized we don't have version catalogs here yet though! #485.

Reply:
Yep.